src/sphmultipcqueue.cpp

```diff
@@ -325,6 +325,7 @@ SPHMPMCQAdvanceHead(SPHLFEntryHeader_t **head_p, sphLFEntryID_t idAlloc, sphLFEn
 	return 0;
 }
 
+#if defined(HAS_GNU_TM) || defined(__HTM__)
```

Comment:
@ThinkOpenly Also we have to call the test sphmultipcqueue_t.c only #if defined(HAS_GNU_TM) || defined(__HTM__)